Pcap++/header/PcapFileDevice.h

```diff
@@ -117,7 +117,7 @@ namespace pcpp
 		 * isn't opened yet, so reading packets will fail. For opening the file call open()
 		 * @param[in] fileName The full path of the file to read
 		 */
-		PcapFileReaderDevice(const std::string& fileName) : IFileReaderDevice(fileName), m_PcapLinkLayerType(LINKTYPE_ETHERNET) {}
+		explicit PcapFileReaderDevice(const std::string& fileName) : IFileReaderDevice(fileName), m_PcapLinkLayerType(LINKTYPE_ETHERNET) {}

```

Comment:
ditto for this c'tor all the classes in this file

Reply:
But I'm not sure about these classes. Since `char*` currently is a valid input. For example https://github.com/seladb/PcapPlusPlus/blob/0b64a41ffcb4a1d7dbb7f1b97a3c84dc24792661/Tests/Pcap%2B%2BTest/Tests/IPFragmentationTests.cpp#L419